deeprank_gnn/models/contact.py

```diff
@@ -53,25 +54,29 @@ def vanderwaals_potential(self) -> float:
 
     @property
     def residue1(self) -> Residue:
-        return self.item1
+        return self.residue1
```

Comment:
I think this is the right attribute (item1 does not exist)

Reply:
I suppose _residue1 will work, but not residue1, since that would be a circular call.

Reply:
In this case either we change `residue1` method name or the attribute name. Anyway, `item1` is not an attribute at all of this class, right? Maybe we can use `item1` and `item2` as attributes names (changing them also in the init method), and leave `residue1` and `residue2` methods?

Reply:
I made Contact inherit from Pair, since it's a pair of residues and their order should be irrelevant. However, Pair has the fields item1 & item2, not residue1 & residue2. An alternative would be not to inherit from Pair and copy some class methods from Pair instead.